### PR TITLE
Normative: Wrap optional _options_ with [ ]

### PR DESCRIPTION
**Pull request by FrankYFTang**

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

**Comment by FrankYFTang** [comment]
try to fix https://github.com/tc39/proposal-temporal/issues/1639

File: spec/duration.html [hunk 1]
```diff
@@ -448,7 +448,7 @@ <h1>Temporal.Duration.prototype.round ( _options_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-temporal.duration.prototype.total">
-      <h1>Temporal.Duration.prototype.total ( _options_ )</h1>
+      <h1>Temporal.Duration.prototype.total ( [ _options_ ] )</h1>
```
Comment (ptomato) [inline spec/duration.html:451]: This one is not optional because the `unit` is required. (Consider adding an `undefined` -> TypeError check in #1646?)

**Review by ptomato — CHANGES_REQUESTED** [review]
This changes the value of the `length` property of the affected functions, could you modify the appropriate tests as well to reflect the new `length` value?

**Comment by ptomato** [comment]
(Converting normative PR to draft so it is not accidentally merged without consensus)

**Comment by ptomato** [comment]
I added tests to this, it should be ready to present for consensus now.

**Review by ljharb — APPROVED** [review]

**Review by ptomato — APPROVED** [review]

**Comment by ptomato** [comment]
This change achieved consensus in TC39.

**Pull request merged**